tests/test_identify_utils.py

```diff
@@ -0,0 +1,89 @@
+"""
+Copyright 2023 Impulse Innovations Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import unittest
+
+from cai_causal_graph import CausalGraph
+from cai_causal_graph.identify_utils import identify_confounders
+from cai_causal_graph.type_definitions import EDGE_T
+
+
+class TestIdentifyConfounders(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
```

Comment:
Suggest adding one test that it works with `TimeSeriesCausalGraph` as well. Should be fine as it is a subclass but worth exercising.